packages/wdio-local-runner/tests/localRunner.test.ts

```diff
@@ -50,7 +95,7 @@ test('should fork a new process', async () => {
         specs: ['/foo/bar.test.js'],
         execArgv: [],
         retries: 0,
-    })
+    } as any)
```

Comment:
Can we not use any and instead use the correct types?

Reply:
These aren't actually required.